tests/spdx/parser/rdf/test_rdf_parser.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import pytest
+
+from spdx.model.document import Document
+from spdx.parser.rdf import rdf_parser
+
+
+def test_rdf_parser_file_not_found():
+    with pytest.raises(FileNotFoundError) as err:
+        wrong_file_path = os.path.join(os.path.dirname(__file__), 'hnjfkjsedhnflsiafg.json')
+        rdf_parser.parse_from_file(wrong_file_path)
+
+    assert err.value.args[1] == "No such file or directory"
```

Comment:
Jep, I didn't use it in the first place as `match` is mapped to `re.search` and this would also pass if the error message would include more than this sentence, but in this case I agree that this doesn't really make a difference.